{"name": "测试验证码错误登录失败", "status": "passed", "attachments": [{"name": "log", "source": "f8ab1bb9-7500-4469-a142-1cbc0c16cbee-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "c3e1bfa4-0192-4fdd-bf87-2da8252492de-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "8dad0a1c-c044-4615-b1af-a30812716008-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "phone", "value": "'15275457999'"}, {"name": "password", "value": "'457888'"}, {"name": "mark_code", "value": "'0000'"}], "start": 1553240748015, "stop": 1553240778428, "uuid": "be1a3543-cb13-44dd-ada0-eaf001da9757", "historyId": "056b3c55ec67c7bd09573ee3dd3fa00c", "fullName": "test_case.test_bpm.test_login.TestLogin#test_login_failed_code_error[15275457999-457888-0000]", "labels": [{"name": "feature", "value": "测试登录"}, {"name": "story", "value": "测试登录失败"}, {"name": "parentSuite", "value": "test_case.test_bpm"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "cadentang"}, {"name": "thread", "value": "15076-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_bpm.test_login"}]}